### PR TITLE
Creating PHP client lib

### PR DESCRIPTION
**Pull request by vcampitelli**
This will be useful for our PHP quickstarts.

### CODE REVIEW

File: php/setup.php [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+            )
+        );
+    }
+
+    private function createApplication(): void
+    {
+        $application = [
+            'name'               => $this->applicationName,
+            'oauthConfiguration' => [
+                'authorizedRedirectURLs' => [$this->applicationBaseUrl],
```
Comment (mooreds) [inline php/setup.php:139]: Is this correct? We redirect to `http://localhost:9011`?

**Review by mooreds — CHANGES_REQUESTED** [review]
Needs a README.md like the other ones. One small comment.

Otherwise looks good.

File: php/setup.php (hunk 1 above)
Comment (vcampitelli) [reply php/setup.php:139]: No, `$this->applicationBaseUrl` is actually `http://localhost:8080` as defined on line 19.

`http://localhost:9011` is actually `$this->baseUrl', which is used to instantiate the `FusionAuthClient` and to set the issuer for the tenant.

Comment (mooreds) [reply php/setup.php:139]: my bad.

**Review by mooreds — APPROVED** [review]
LGTM

**Pull request merged**